src/utils/formatRunLogTimestamp.ts

```diff
@@ -0,0 +1,7 @@
+// converts a runlog's Timestamp to a formatted string, used in displaying run logs
+import { fromUnixTime, format } from 'date-fns'
+
+export default function formatRunLogTimestamp (t: number): string {
```

Comment:
you can import the util function `toMilliFromNano` & use it to convert the timestamp from nanoseconds to milliseconds (which works with the node `Date` function):

`format(new Date(toMilliFromNano(t))), "yyyy-MM-dd'T'HH:mm:ss.SSSxxx")`